src/redux/books/booksSlice.js

```diff
@@ -1,18 +1,38 @@
+import axios from 'axios';
 import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
-
-import { fetchBooks, addBook, removeBook } from '../../AddApi/Api';
-
-const initialState = {
-  books: [],
-  status: 'idle',
-  error: null,
-};
-
+import { fetchBookApi, addBook, removeBook } from '../../BooksApi/BooksApi';
+
+const initialState = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  },
+];
+export const getbooks = createAsyncThunk(
+  'books/getbooks',
+  async () => {
+    const response = await axios.get('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/Oc9abMsHOhG37sSkzGFr/books/');
+    return response.data;
+  },
+);
 export const fetchBooksAsync = createAsyncThunk('books/fetchBooks', async () => {
```

Comment:
- [ ] Kindly make sure that fetchBooksAsync actually updates the state of booksSlice. Right now I'm still unable to add a book. Only the default books from the books array are displayed. Kindly make sure it is possible to add a book.

[Screencast from 06-15-2023 02:56:32 PM.webm](https://github.com/theresetuyi/React-Bookstore/assets/63087491/24c9eeac-1688-4f02-a3d4-4e39ecaa730d)